java/k8s-addons/src/main/java/com/instaclustr/cassandra/k8s/SeedProvider.java

```diff
@@ -1,14 +1,14 @@
 package com.instaclustr.cassandra.k8s;
 
-import com.google.common.collect.ImmutableList;
```

Comment:
This comment is non-blocking on the PR and doesn't need to get fixed, but for the future, try not to check in unrelated changes to the feature branch. Especially if the IDE is just redoing imports :)